thirdparty/delay/delay.go

```diff
@@ -44,3 +43,53 @@ func (d *delay) Get() time.Duration {
 	defer d.l.Unlock()
 	return d.t
 }
+
+// VariableNormal is a delay following a normal distribution
+// Notice that to implement the D interface Set can only change the mean delay
+// the standard deviation is set only at initialization
+func VariableNormal(t, std time.Duration) D {
+	v := &variableNormal{
+		std: std,
+		rng: rand.New(rand.NewSource(int64(time.Now().Nanosecond()))),
```

Comment:
Using one random source per one delay instance runs risk of it being used just one time.
Pseudo Random Generators used just one time will not generate uniformly distributed random numbers.

I would recommend using one package wide random generator.